app/controllers/account/comments_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Account::CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @comment.assign_attributes(tenant_id: current_tenant.id)
+    if @comment.save
+      respond_to_format
+    else
+      respond_to_format(render(js: "alert('Comment is invalid!');"))
+    end
+  end
+
+  def destroy
+    @comment = resource
+    if current_user.privileged? && current_tenant_comment?
+      @comment.destroy
+      respond_to_format
+    else
+      respond_to_format(render(js: "alert('No rights to delete this comment');"))
+    end
+  end
+
+  private
+
+  def respond_to_format(block = nil)
+    respond_to { |format| format.js { block } }
+  end
+
+  def current_tenant_comment?
+    resource.tenant_id == current_tenant.id
+  end
+
+  def resource
+    Comment.find(params[:id])
```

Comment:
Code must be
current_tenant.comments.find

method current_tenant_comment? must be deleted